Source/NSTableView.m

```diff
@@ -3300,80 +3310,87 @@ - (void) validateEditing
       NSString *string;
       id newObjectValue = nil;
       BOOL validatedOK = YES;
+      GSKeyValueBinding *theBinding = nil;
```

Comment:
I would prefer two definitions closer to the usages.

Reply:
Done.